devito/symbolics/extended_sympy.py

```diff
@@ -303,9 +304,9 @@ def base(self):
 
     def __str__(self):
         if self.base.is_Symbol:
-            return "&%s" % self.base
+            return "&%s" % ccode(self.base)
```

Comment:
rebase ?

Reply:
I think I didn't understand. 

I'll do a rebase, but this part of the code is correct, the `ccode` need to be inserted here.

Reply:
never mind, ignore me: I thought for a moment that this had already been fixed in master and that somewhat you were missing a rebase